README.md

```diff
@@ -46,7 +46,7 @@ the unit tests) before expecting it to work! For a working example, see `example
 There are several steps involved to execute CQL.  First, you must create a JSON representation of
 the ELM. For the easiest integration, we will generate a JSON file using cql-to-elm:
 
-1. Install the [Java 8 SDK](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
+1. Install the [Java 11 SDK](https://github.com/AdoptOpenJDK/homebrew-openjdk)
```

Comment:
Since homebrew is Mac-specific I think we need to either link to the main (OS-agnostic) AdoptOpenJDK page or have two links (Windows and Max).

Reply:
Done!